src/tensor_ops.jl

```diff
@@ -152,17 +157,22 @@ Computes the trace of a second or fourth order tensor.
     @code :(return s)
 end
 
+const vol = trace
```

Comment:
Yes I think vol should only be used on second order tensors.

`vol(S::SecondOrderTensor) = trace(S)` should do it.

`vol` should probably be exported as well